templates/idx.html

```diff
@@ -1048,6 +1048,7 @@ <h3 class="title is-3">
                     var hostPort = map[0];
                     var containerPort = map[1][0];
                     var containerUrl = map[1][1];
+                    containerUrl.replace(containerPort.toString(), hostPort.toString());
```

Comment:
I think we need a `containerUrl = containerUrl.replace(...);`.
Otherwise, the replaced string is just tossed.

Reply:
Updated.